docker-compose.yml

```diff
@@ -34,7 +34,7 @@ services:
     stdin_open: true
     tty: true
   nginx:
-    image: nginx:1.19
+    image: nginx:1.19.10-alpine
```

Comment:
We use `ubuntu` / `debian` in prod, any change of using either of them here?

Reply:
> We use ubuntu / debian in prod, any chance of using either of them here?

Yep. Although it's much bigger, I've used `nginx:1.20` which uses `Debian` as the base image.